base/doc/ltnews37.tex

```diff
@@ -159,6 +159,41 @@ \subsection{\pkg{doc}: Index \texttt{\textbackslash\textvisiblespace} correctly}
 \githubissue{943}
 
 
+\subsection{\pkg{doc}: Support the \pkg{upquote} package}
+
+The default quote and backquote characters in typewriter fonts are
+typographical quotes, e.g., the input
+\begin{verbatim}
+   verb*/`prog 'my input'`/
```

Comment:
`verb*/`prog 'my input'`/`
misses the leading backslash

    `\verb*/`prog 'my input'`/`

Reply:
thanks